rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -613,19 +697,11 @@ def _get_staking_events_graph(
         }
         events = []
         while True:
-            try:
-                result = self.graph.query(  # type: ignore # caller already checks
-                    querystr=querystr,
-                    param_types=param_types,
-                    param_values=param_values,
-                )
-            except RemoteError as e:
-                msg = str(e)
-                self.msg_aggregator.add_error(
-                    f'{msg}. All AdEx balances queries are not functioning until this is fixed.',
-                )
-                raise
-
+            result = self.graph.query(  # type: ignore # caller already checks
```

Comment:
why did you remove exception handling here?

Reply:
It was after talking about the role of message aggregator and PnL report. I wasn't sure about how useful the message was. I've put it back.

Reply:
I am not saying that what you did was wrong. I am just asking to understand the reasoning and where is the exception going to be handled.